core/shared/utils/bh_atomic.h

```diff
@@ -6,6 +6,8 @@
 #ifndef _BH_ATOMIC_H
 #define _BH_ATOMIC_H
 
+#include <stdint.h>
```

Comment:
Could you include `bh_platform.h` instead? like other files of this folder.

Reply:
Done, thank you!